CDCgov/prime-reportstream
# KS Onboarding - Build Receiver Settings

This PR creates an initial setting for KS receiver.

Test Steps:
1. clone the repos
2. Download the pdi-covid-19-ks.csv file into junk
3. reloadsettings
4. run: curl -X POST -H "client:simple_report" -H "content-type:text/csv" --data-binary "@./junk/pdi-covid-19-ks.csv" "http://localhost:7071/api/reports"
5. There is no error in the response
6. Verify the file in prime-router/build/sftp is there

Test file: 
[pdi-covid-19-ks.csv](https://github.com/CDCgov/prime-reportstream/files/11191803/pdi-covid-19-ks.csv)


## Changes
- *Include a comprehensive list of changes in this PR*
- *(For web UI changes) Include screenshots/video of changes*

## Checklist

### Testing

### Process

## Linked Issues
- Fixes #issue
- https://github.com/CDCgov/prime-reportstream/issues/9044

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
